internal/controllers/operator_controller_test.go

```diff
@@ -1061,8 +1061,7 @@ var _ = Describe("Operator Controller Test", func() {
 
 func verifyInvariants(ctx context.Context, c client.Client, op *operatorsv1alpha1.Operator) {
 	key := client.ObjectKeyFromObject(op)
-	err := c.Get(ctx, key, op)
-	Expect(err).To(BeNil())
+	Expect(c.Get(ctx, key, op)).To(Succeed())
```

Comment:
As far as I understand `Expect(funcCall()).To(Succeed())` and `Expect(err).NotTo(HaveOccurred())` do the same thing, but the first one is supposed to be used with functions which only return one `error`. I personally not too concerned about using one style over another. Important one is to use one of these styles over`Expect(err).To(BeNil())` because they give better failure messages.

In general, I'm not a big fan of Ginkgo & Gomega because there are usually 10 ways to do the same.